packages/blade/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,54 @@
+import type { ReactElement } from 'react';
+
+import type { IconComponent } from '..';
+import StyledIconButton from './StyledIconButton';
+
+export type IconButtonProps = {
```

Comment:
how are we deciding the color?

Reply:
Discussing [above](https://github.com/razorpay/blade/pull/613#discussion_r944196472)